src/forms/TextInput.jsx

```diff
@@ -73,7 +73,6 @@ const TextInput = (props) => {
 					placeholder={placeholder}
 					className={classNames}
 					onChange={onChange}
-					maxLength={maxLength}
```

Comment:
Just to be clear, we still want to support `maxLength` as a prop, but not enforce it on the input element itself? 

Could be confusing for a future dev to encounter, so I wouldn't mind seeing this behavior commented. Looks fine to me otherwise.

Reply:
exactly. this is the way textarea behaves and design wants it consistent.